### ISSUE TITLE
support inline '< >' symbols

### ISSUE THREAD

**Issue by avalanche1** [dalgard/meteor-jade#7]
Hi!
Could you make [this](https://forums.meteor.com/t/viewmodel-2-a-new-level-of-simplicity/13581/134) work please?

**Comment by dalgard** [comment]
The bind expression inside the string isn't handled by the `jade` package, it's simply passed to the `b` bind helper – sorry.

I would generally recommend keeping logic out of the templates and using the viewmodel instead.

**Comment by avalanche1** [comment]
Yeah, I agree. Just thought that if html supports this, then jade should as well.

**Comment by dalgard** [comment]
Oh. I see that I was too quick to judge – the problem is with the compiler, of course.

Does it work if you use the `!=` syntax described [here](http://jade-lang.com/reference/attributes/#unescaped)?

**Comment by brajt** [comment]
It doesn't. With the code being:

```
template(name='listUserGamesBox')
  +box
    p($b!="if:2 > 3") ok1
    p($b!="if:2 < 3") ok2
    p($b!="if:3 > 2") ok3
    p($b!="if:3 < 2") ok4
```

The error message is still:

```
While building package brajt:rpg-games:
   user/listUserGames.jade: Jade syntax error: Expected tag name after `<`
   {{{b "if:2 < 3"}}}

```
